applications/salk-portal/package-lock.json

```diff
@@ -8968,41 +8968,6 @@
       "resolved": "https://registry.npmjs.org/material-colors/-/material-colors-1.2.6.tgz",
       "integrity": "sha512-6qE4B9deFBIa9YSpOc9O0Sgc43zTeVYbgDT5veRKSlB2+ZuHNoVVxA1L/ckMUayV9Ay9y7Z/SZCLcGteW9i7bg=="
     },
-    "material-ui-dropzone": {
```

Comment:
@paularah please remove this file, package-lock should not be committed and pushed